ServerRoomMonitoringGenerator/Program.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using RabbitMQ.Client;
+using System.Text;
+
+class Send
+{
+    public static void Main()
+    {
+        var factory = new ConnectionFactory() { HostName = "localhost" };
```

Comment:
Lines 9-27 look like the first method for something like QueueRepository which can be injected.
It would be responsible for giving access to the Rabbit queue and expose the method `void SendMessage(string message)`.